cmd/dataapi/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Dockerfile References: https://docs.docker.com/engine/reference/builder/
+
+# Start from the latest golang base image
+FROM golang:latest
+
+# Add Maintainer Info
+LABEL maintainer="Katchoua <webmaster@chnm.gmu.edu>"
+
+# Set the Current Working Directory inside the container
+WORKDIR /app
+
+# Copy the source from the current directory to the Working Directory inside the container
+COPY ./../../ .
+
+# Build the Go app
+RUN cd cmd/dataapi && go build -o main .
```

Comment:
@katchoua Is there a particular reason you want to rename the binary from `dataapi` to `main`? Not a big deal one way or the other.

I need to investigate whether it is necessary to also explicitly compile the library, to make sure it is up to date when the CLI is rebuilt, or not. I will look into best practices for Go and Docker.